upgrade_test.py

```diff
@@ -45,14 +47,21 @@ def truncate_entries(func):
     @wraps(func)
     def inner(self, *args, **kwargs):
         node = args[0]
-        with self.db_cluster.cql_connection_patient(node, keyspace='truncate_ks') as session:
-            self.cql_truncate_simple_tables(session=session, rows=NUMBER_OF_ROWS_FOR_TRUNCATE_TEST)
+        with self.db_cluster.cql_connection_patient(node, keyspace='truncate_ks', connect_timeout=240) as session:
```

Comment:
shouldn't this timeout be a bit bigger than the DML command?

Reply:
increased to 300